internal/pkg/controller/http/common.go

```diff
@@ -37,29 +36,29 @@ func NewV2CommonController(dic *di.Container) *V2CommonController {
 // Ping handles the request to /ping endpoint. Is used to test if the service is working
 // It returns a response as specified by the V2 API swagger in openapi/v2
 func (c *V2CommonController) Ping(writer http.ResponseWriter, request *http.Request) {
-	response := common.NewPingResponse()
-	c.sendResponse(writer, request, contractsV2.ApiPingRoute, response, http.StatusOK)
+	response := commonDTO.NewPingResponse()
+	c.sendResponse(writer, request, common.ApiPingRoute, response, http.StatusOK)
 }
 
 // Version handles the request to /version endpoint. Is used to request the service's versions
 // It returns a response as specified by the V2 API swagger in openapi/v2
 func (c *V2CommonController) Version(writer http.ResponseWriter, request *http.Request) {
```

Comment:
Rename to controller to remove `V2` from name

Reply:
Renamed the controller.